acquia_lift.services.yml

```diff
@@ -11,6 +11,9 @@ services:
   acquia_lift.service.helper.help_message_helper:
     class: Drupal\acquia_lift\Service\Helper\HelpMessageHelper
     arguments: ['@config.factory', '@link_generator']
+  acquia_lift.service.helper.lift_api_helper:
+    class: Drupal\acquia_lift\Service\Helper\LiftAPIHelper
+    arguments: ['@config.factory']
```

Comment:
In my past, API and RESTful helpers often spin into much larger local but contained complexity as project grows. I have made a habit of starting them off with their distinct Directory, say "Api" or "rest". I think I sometimes observe other repos do this to, so I would recommend that here.

But if you'd like to keep as-is until we actually get to refactor and split when complex, that's good too.